Source/Scene/PointCloud3DTileContent.js

```diff
@@ -781,8 +785,11 @@ define([
         var hasNormals = content._hasNormals;
         var hasBatchIds = content._hasBatchIds;
         var backFaceCulling = content._backFaceCulling;
+        var normalShading = content._normalShading;
         var vertexArray = content._drawCommand.vertexArray;
 
+        var normalsEnabled = hasNormals && normalShading;
```

Comment:
Yeah, except for the part that actually does normal shading.

Another approach is to continue to use `normalsEnabled` but set it equal to `normalShading || backFaceCulling`.